tgui/packages/tgui/interfaces/NtosRecords.jsx

```diff
@@ -55,18 +55,18 @@ export const NtosRecords = (props) => {
                 {record.name}
               </Box>
               <br />
-              Rank: {record.rank}
+              Должность: {record.rank}
               <br />
-              Species: {record.species}
+              Вид: {record.species}
               <br />
-              Gender: {record.gender}
+              Гендер: {record.gender}
```

Comment:
```suggestion
              Пол: {record.gender}
```
хз, вроде нет разделения гендеров, только пол?

Reply:
там нейтральный есть, так что лучше гендер

Reply:
Нейтральный - скорее не гендер, а без первичных признаков какого-либо рода. Будущее, выращенный клон